src/objects/DataImportBatch__c.object

```diff
@@ -430,6 +430,36 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>Status__c</fullName>
```

Comment:
```suggestion
        <fullName>Batch_Status__c</fullName>
```

Reply:
@vmartinov-sforg Would having this field name more clearly call out that it's the batch's status make reporting easier for customers?

Reply:
Done.